### DISCUSSION TITLE
[Issue] Package try to update a slug while updating a non-related field

### DISCUSSION THREAD

**Discussion by nikrovir** [spatie/laravel-sluggable#258] [Q&A]
Configuration example:
```php
public function getSlugOptions() : SlugOptions
{
    return SlugOptions::create()
        ->generateSlugsFrom(['title'])
        ->saveSlugsTo('slug')
        ->usingSeparator('-')
        ->slugsShouldBeNoLongerThan(200);
}
```
Code example:
```php
$model = MyModel::select(['id'])->findOrFail($someID);

$model->update([
    'order' => 5000,
]);
```
This code example attempts to update a slug by fetching only the ID from the database and without interacting with the field that the slug is generated from.

As a result of executing this code, we will get a slug of the form "-1".

It seems to me, or is this not correct behavior? Perhaps there is already a ready solution to the problem (I would not want to add "title" to the selection)?

**Comment by sohailahmad07** [comment]
I am also have the same problem did you find a solution

**Reply by nikrovir** [reply]
No solution has been found yet
